addons/html_builder/static/src/sidebar/block_tab.js

```diff
@@ -148,7 +148,7 @@ export class BlockTab extends Component {
         });
 
         if (selectedSnippetEl) {
-            await scrollTo(selectedSnippetEl, { extraOffset: 50 });
+            scrollTo(selectedSnippetEl, { extraOffset: 50 });
```

Comment:
In the past this was a real desire to wait for the scroll: `onBuilt` could depend on it (although that one is probably not a good idea) but mainly the `PublicWidget` activation definitely depends on it. Now, I don't know how Interaction are started after drop in the new builder, maybe it is ok :shrug: